config/defaults.yml

```diff
@@ -13,3 +13,4 @@ ec2_ssh_user: "ec2-user"
 cluster_version: "{{ lookup('env', 'CLUSTER_VERSION') or '4'}}"
 cluster_name: "{{ lookup('env', 'CLUSTER_NAME') or 'jenkins-ci'}}"
 oc_binary_location: "{{ ansible_env.HOME }}/bin/"
+ocp_version: "{{ lookup('env', 'OPENSHIFT_VERSION') }}"
```

Comment:
We should decide, where the variable should be placed, now it is redundant, because of
https://github.com/fusor/mig-ci/blob/bb9a1ae50a1ca78ba68e2dc65adf4fa25adda78f/roles/ocp3_aws_deploy/defaults/main.yml#L1